### PR TITLE
ensure nice error message when non-existing path is passed to apply_regex_substitutions

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86 please review

File: easybuild/tools/filetools.py [hunk 1]
```diff
@@ -741,6 +741,9 @@ def apply_regex_substitutions(path, regex_subs):
     else:
         _log.debug("Applying following regex substitutions to %s: %s", path, regex_subs)
 
+        if not os.path.exists(path):
```
Comment (wpoely86) [inline easybuild/tools/filetools.py]: why not wrap in a try? The file can exists without proper permissions?

Comment (boegel) [inline easybuild/tools/filetools.py]: yeah, ok, will do

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**